bbq/javatests/java/com/google/bbq/QueryResponseTest.java

```diff
@@ -26,53 +27,59 @@
 import java.security.SecureRandom;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
 
 /**
  * Battery of tests for the QueryResponse
  */
+@RunWith(RobolectricTestRunner.class)
+@Config(manifest = Config.NONE)
 public class QueryResponseTest {
-    @Mock
-    private Parcel mockParcel;
 
-    private QueryResponse underTest;
-    private String mResponderPackage = "mResponderPackage";
-    private long mResponseId = 128L;
-    private byte[] mResponseMessage = new byte[64];
+    private static final String PACKAGE = "com.example.provider";
+    private static final long RESPONSE_ID = 1234L;
+    private static final byte[] RESPONSE_MESSAGE = new byte[] { 0, 1, 2, 3, 4 };
 
-    @Before
-    public void setUp() throws Exception {
-        MockitoAnnotations.initMocks(this);
-        SecureRandom sr = new SecureRandom();
-        sr.nextBytes(mResponseMessage);
-
-        underTest = new QueryResponse(mResponderPackage, mResponseId, mResponseMessage);
-    }
+    private static final QueryResponse QUERY_RESPONSE =
+            new QueryResponse(PACKAGE, RESPONSE_ID, RESPONSE_MESSAGE);
 
     @Test
     public void testDescribeContents() throws Exception {
-        assertEquals(0, underTest.describeContents());
+        assertEquals(0, QUERY_RESPONSE.describeContents());
```

Comment:
Nit: Use of old test assertions style, would you mind switching it to assertThat() style?